docs/apache-airflow/concepts/dags.rst

```diff
@@ -564,7 +564,7 @@ doc_md      markdown
 doc_rst     reStructuredText
 ==========  ================
 
-Please note that for DAGs, ``doc_md`` is the only attribute interpreted.
+Please note that for DAGs, ``doc_md`` is the only attribute interpreted and it can also contain the path of a markdown file containing the DAG's documentation.
```

Comment:
This should also document where the Markdown file would be found (i.e. if a relative path is supplied, where would the path be relative to?)

Reply:
Rewrote it, similarly to the `sql` param in the SQL operators, also referencing the relative path